cea/interfaces/dashboard/lib/auth/providers.py

```
@@ -16,85 +15,134 @@ class StackAuth:
-            payload = jwt.decode(
+            return jwt.decode(
                 self.access_token,
                 signing_key,
                 algorithms=["ES256"],
                 audience=self.project_id,
             )
-
-            return payload
-
+        except jwt.ExpiredSignatureError:
+            raise CEAAuthError(f"Token expired")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)